lib/compat/wordpress-6.2/default-filters.php

```diff
@@ -18,7 +18,9 @@
  */
 
 add_action( 'switch_theme', 'wp_theme_has_theme_json_clean_cache' );
+add_action( 'switch_theme', array( 'WP_Theme_JSON_Resolver_Gutenberg', 'clean_cached_data' ) );
```

Comment:
These were missing, causing some tests to fail. As long as we maintain a `WP_Theme_JSON_Resolver_Gutenberg` class, we need them.

Reply:
Exactly.
Tests often help to identify these little imperfections.